internal/lsp/lsp.go

```diff
@@ -111,6 +111,7 @@ type DidCloseTextDocumentParams struct {
 type CompletionParams struct {
 	TextDocumentPositionParams
 	CompletionContext CompletionContext `json:"contentChanges"`
+	LowercaseKeywords bool
```

Comment:
This struct must be in accordance with the lsp specification, so adding your own parameters is no longer desirable.